scrape.py

```diff
@@ -17,6 +17,7 @@
 from icalendar import Calendar, Event
 import logbook
 import requests
+from socket import gethostname
```

Comment:
`socket` is part of the stdlib, so this import belongs to the first import block. (While you're at it, there should be just one line of whitespace between the two import blocks, not 2 - could you get rid of one?)